### PR TITLE
Refactor/game teams gv

### PR DESCRIPTION
**Pull request by GVarenhorst**
- Update line spacing and pagination for game_teams lib and spec files
- Change lines to match convention order with other spec tests
- Correct README so that table of contents follows order, correct pagination

### CODE REVIEW

**Review by dani-wilson — APPROVED** [review]
looking spick and span

**Comment by DChavez18** [comment]
Thanks for tightening up the readme file!

**Comment by algibson1** [comment]
I can't comment directly on it since it looks like it was added in a previous pull and not this one, but there's a "back to top" on lines 25 and 48 (in addition to the one at the bottom)
They seem random since we aren't adding them to every section. Does anyone else have thoughts on those?

File: README.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 ## Table of Contents
   - [Check-ins](#check-ins)
-  - [Approach To Code Design](#plan-for-project-organization)
-  - [Organization](#approaches-to-project-organization)
+  - [Organization](#plan-for-project-organization)
```
Comment (algibson1) [inline README.md:5]: Our approaches to organization section still isn't accounted for- unless we wanted to group it up with the plan?
In which case, can we change `Organization` to `Plan and Approaches To Organization`?

Comment (algibson1) [reply README.md:5]: Or change the headline of `Approaches to Project Organization` to a subheading?

**Pull request merged**